tox.ini

```diff
@@ -1,19 +1,24 @@
 [tox]
-envlist = py{26,27,34,35,36},linting
+envlist=
+    py{26,27,34}-defaultreactor
+    py{35,36}-{default,qt5}reactor
+    linting
 
 
 [testenv]
-deps=pytest
-     greenlet
-     twisted
-commands=py.test []
-sitepackages=False
-
-[testenv:py26]
-deps=pytest<3.3
-     greenlet
-     twisted<15.5
-commands=py.test []
+deps=
+    greenlet
+    py26: pytest<3.3
+    py{27,34,35,36}: pytest
+    py26: twisted<15.5
+    py{27,34,35,36}: twisted
+    qt5reactor: pytest-qt
+    qt5reactor: qt5reactor
+    qt5reactor: pytest-xvfb
+    qt5reactor: pyqt5
+commands=
+    defaultreactor: py.test --reactor=default []
+    qt5reactor: py.test --reactor=qt5reactor []
```

Comment:
I don't like this (which works) or the previous `setenv` approach (which almost worked except for on AppVeyor with `TOXENV`).  Haven't found something nicer though since I could

```ini
reactor_option=
    defaultreactor: default
    qt5reactor: qt5reactor
commands=py.test --reactor={reactor_option} []
```
but I don't know how to use that in a substitution in `commands=` and just get

```
tox.ConfigError: ConfigError: substitution key 'reactor_option' not found
```